lib/config/graphite.go

```diff
@@ -0,0 +1,288 @@
+package config
```

Comment:
the new file should add the Apache License title:
```
/*
Copyright openGemini authors.
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
 http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
```
and also , if you copy some code from
```
https://github.com/influxdata/influxdb/tree/1.8/services/graphite
https://docs.influxdata.com/influxdb/v1/supported_protocols/graphite/
```
you should declare the source and copyright of the code, like this:
```
/*
Copyright (c) 2013-2016 Errplane Inc.
This code is originally from: https://github.com/influxdata/influxdb/blob/v0.10.3/services/meta/store_fsm.go
*/
```
ref: app/ts-meta/meta/snapshot.go